src/Behat/Behat/Tester/Result/ExecutedStepResult.php

```diff
@@ -88,7 +88,7 @@ public function getException()
     }
 
     /**
-     * {@inheritdoc}
+     * @return self::PENDING | self::FAILED | self::PASSED
```

Comment:
We need to be more strict about the range of TestResult values that can be returned or accepted by the different functions